src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    private String nameOfMachine3;
+
+    @Override
+    public void setName(String nameOfMachine3) {
+        super.setName(nameOfMachine3);
+    }
+
+    @Override
+    public String getNameOfMachine() {
+        return super.getNameOfMachine();
+    }

```

Comment:
Getters/setters are not required to be overridden.
In this task properties for classes are not required, please remove them